tests/unittests/tests-hashes/tests-hashes-keccak.c

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright (C) 2017 Adrian Herrmann <adrian.herrmann@fu-berlin.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     unittests
+ * @{
+ *
+ * @file
+ * @brief       Test cases for the Keccak hashing package
+ *
+ * @author      Adrian Herrmann <adrian.herrmann@fu-berlin.de>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdint.h>
+
+#include "KeccakHash800.h"
+#include "KeccakHash1600.h"
+#include "embUnit.h"
+
+Keccak800Hash_instance* hash800Instance;
+Keccak1600Hash_instance* hash1600Instance;
+
+const BitLength databitlen = 80;
+const char datastring[] = "0123456789";
+const BitSequence* data = (unsigned char*) datastring;
+
+const char hashvalstring800[] =
+            "a6305bbe48f1f8d2c58dfd9731974fe85321c09cba8a944b0635a9ba07443324";
+BitSequence* hashval800;
+
+const char hashvalstring1600[] =
+            "8f8eaad16cbf8722a2165b660d47fcfd8496a41c611da758f3bb70f809f01ee3";
+BitSequence* hashval1600;
+
+static void setUp(void)
+{
+    /* Initialize */
+    hash800Instance = malloc(sizeof(Keccak800Hash_instance));
+    hashval800 = malloc(32);
```

Comment:
In RIOT we like to avoid malloc as much as possible.
I suggest to statically allocate these variables, and to remove setUp and Teardown 
(like done in other hash tests such as [here](https://github.com/RIOT-OS/RIOT/blob/master/tests/unittests/tests-hashes/tests-hashes-sha256.c))